### ISSUE TITLE
Adjust the default branch after a major release

### ISSUE THREAD

**Issue by leandro-lucarella-sociomantic** [sociomantic-tsunami/neptune#77]
One step that is annoying and easy to forget and prone to errors is to update the GitHub project default branch after a major release. This could eventually be automatized too by the release tool.

**Comment by mathias-lang-sociomantic** [comment]
But with neptune the default branch stays the same for X months after a new major, so it's not clear to me what the issue is ?

**Comment by leandro-lucarella-sociomantic** [comment]
Just forgetting to update it (and having to manually do so).

**Comment by gavin-norman-sociomantic** [comment]
I don't get it. Making a major release should not change the default branch. The default branch should move when the current default branch goes out of support. Or am I missing something?

**Comment by leandro-lucarella-sociomantic** [comment]
Right, no, I mixed things up. So automating this is not that easy them :(

Still I think it would be nice to have some automation for when a major goes out of support, so it's just one command, but them maybe it should be a different tool, like `neptune-eol" (end of life).

I'll rename the issue.
